.github/workflows/main.yml

```diff
@@ -34,7 +37,7 @@ jobs:
     if: github.ref == 'refs/heads/master'
     strategy:
       matrix:
-        python-version: ["3.7", "3.8"]
+        python-version: ["3.8"]
```

Comment:
Are we sure we want to remove 3.7 tests ?

Reply:
The python version of line 40 was only used to publish on PyPI the new version of the module. So, if the module is published twice (i.e. with both python versions), there is, unfortunately, an error, that's why I removed it. Did I correctly understand how the workflows worked?

Reply:
Ah I see! In that case let's not use a matrix to choose python versions but smthg like this: https://github.com/OpenMined/PySyft/blob/master/.github/workflows/publish.yml#L12
(just for the python version selection)

Reply:
I changed it. Are the changes right for you?